Cargo.toml

```diff
@@ -8,6 +8,7 @@ publish = false
 [dependencies]
 anyhow = "1.0.86"
 argh = "0.1.12"
+byte_string = "1.0.0"
```

Comment:
I would prefer to not use this. Would [it help to ](https://docs.rs/imap-types/latest/imap_types/utils/fn.escape_byte_string.html) help you?

(I always wanted to remove the `utils` module but it's just too useful... Feature creep...)

Reply:
Sure, I forgot its existence.